recovery/cloudbuild_ci.yaml

```diff
@@ -153,9 +153,9 @@ steps:
           \"bucket\": \"${_LOG_NAME}\",
           \"kmsKeyName\": \"ft-log-ci\",
           \"kmsKeyRing\": \"firmware-release-ci\",
-          \"kmsKeyVersion\": ${_KEY_VERSION},
+          \"kmsKeyVersion\": '${_KEY_VERSION}',
```

Comment:
could this use '"' for consistency, or does it need to be the single quote? (same elsewhere)